gap/main.g

```diff
@@ -580,9 +580,17 @@ Chief := function( galoisField,mat,a,b,IsHPC,withTrafo )
         D := TransposedMat( GAUSS_RRF( galoisField,X,
             TransposedMat( GAUSS_CEX( galoisField,v,D )[1] ),v ) );
     fi;
-    
 
+    
     heads := GAUSS_createHeads(v, w, DimensionsMat(mat)[2]);
+
+    if verify and not IsMatrixInRREF(Concatenation(-B, D) * mat) then
```

Comment:
Do we assume that `withTrafo` is set when `verify` is set? Should we maybe declare something like the following in the beginning of the function Chief:
```
if verify then
  withTrafo := true;
fi;
```

Reply:
`withTrafo` is a parameter of `Chief` so it is always defined.

Or do you mean that it should always be true when `verify` is true? That is the case. `Chief` and `DoEchelon..` are going to be merged once we're done with #170, see #86, so we can assume for now that `Chief` is only called through `DoEchelon..`. The latter function makes sure that `verify` is only true if `withTrafo` is also true.